### PR TITLE
Enabled passing kwargs to export()

### PR DESCRIPTION
**Pull request by borisfom**
Also, removed previous (non–functional) extension for dicts in export - this schema matches what onnx.export expects.
 
Signed-off-by: Boris Fomitchev <bfomitchev@nvidia.com>

### CODE REVIEW

**Review by okuchaiev — APPROVED** [review]
exortable changes look good to me. @blisc what about tts changes?

**Comment by borisfom** [comment]
> exortable changes look good to me. @blisc what about tts changes?

tts changes are to conform to new extended syntax accepted by onnx.export (optional Dict as the last input tuple element, representing kwargs) instead of former non-standard extended syntax I briefly introduced in previous revision (accepting a Dict instead of a tuple as inputs, not really used except in this test example).

File: tests/collections/tts/test_waveglow.py [hunk 1]
```diff
@@ -54,7 +54,10 @@
 def input_example(sz):
```
Comment (blisc) [inline tests/collections/tts/test_waveglow.py:54]: Can we undo the changes to this file?

Comment (borisfom) [reply tests/collections/tts/test_waveglow.py:54]: Why, it would fail to compile then? See my comment above.
